### DISCUSSION TITLE
Dokcer Image: RoboticsAcademy vs RoboticsBackend

### DISCUSSION THREAD

**Discussion by codezerro** [JdeRobot/RoboticsAcademy#2694] [Q&A] [answered]
Whats the difference between jderobot/robotics-academy and jderobot/robotics-backend Docker image?
Does jderobot/robotics-academy support all of the exercises that the jderobot/robotics-backend image has?

**Comment by jmplaza** [comment, marked as answer]
Hi @codezerro, 

RoboticsAcademy is the repository with the source code of all the exercises and the main Django webserver. For all of them the execution requires the RoboticsBackend Docker image to be launched. 

The RoboticsBackend docker image includes (a) the Robotics Infrastructure (ROS, Gazebo, and common robotics dependencies), (b) the Robotics Application Manager, and (c) the RoboticsAcademy (a certain snapshot of the repo). 

Hope it helps,
